extras/zot/enable-zot/enable-zot.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: zot-cache-enable
+spec:
+  rules:
+  - name: clusterconfig
+    context: 
+       - name: userconfig
+         configMap:
+           name: "{{ request.object.spec.userConfig.configMap.name }}"
+           namespace: "{{ request.object.spec.userConfig.configMap.namespace }}"
+    match:
+      any:
+      - resources:
+          kinds:
+            - App
+          namespaces:
+            - 'org-*'
+          operations:
+            - CREATE
```

Comment:
This means it will work for new cluster only, right? What about updates to those clusters? I mean what if someone / some automation removes the configmap reference? Same for the configmap creation below? Do we want to enforce the state of the reference and the contents of the CM? Tho I am not sure how the `/spec/extraConfigs/-` would handle it, like keep adding a lot of entries to the map.

Reply:
@uvegla as I read through the policy, I think it's fine as it's only acting on App Create so the `extraConfigs` entry will only be added once.